sdk/storage/Azure.Storage.Queues/src/Models/QueueTraits.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+
+namespace Azure.Storage.Queues.Models
+{
+    /// <summary>
+    /// Specifies options for listing queues with the
+    /// <see cref="QueueServiceClient.GetQueuesAsync"/> operation.
+    /// </summary>
+    [Flags]
+    public enum QueueTraits
+    {
+        /// <summary>
+        /// Default flag specifying that no flags are set in <see cref="QueueTraits"/>.
```

Comment:
Nit - let's change this to something more like "Flag specifying only the default information is returned when listing QueueItems via GetQueues and GetQueuesAsync"

Reply:
I have the operation info in the Enum documentation, but I updated to:
Flag specifying only the default information for queues should be included.